.github/workflows/ci.yml

```diff
@@ -556,6 +556,63 @@ jobs:
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
 
+  rdbms-integration-tests:
+    if: needs.detect-changes.outputs.rdbms-integration-tests == 'true'
+    needs: [detect-changes]
+    name: "[IT] RDBMS"
+    timeout-minutes: 15
```

Comment:
We should aim to reduce this value, if possible, to what is actually needed. The standalone job took ~6 minutes.

You can check execution times of the new job on this PR and adjust based on that (round up to nearest 5/10 minutes).

Reply:
Looking at https://github.com/camunda/camunda/actions/runs/14999255846/job/42141544742?pr=31948 the job seems to need close to 5 minutes already. Let's pick a slightly higher value to avoid premature timeouts, maybe 7mins.